apm-agent-plugins/apm-logging-plugin/apm-log4j2-plugin/src/test/java/co/elastic/apm/agent/log4j2/Log4j2InstrumentationTest.java

```diff
@@ -92,7 +96,10 @@ public Log4j2LoggerFacade() {
         @Override
         public void open() {
             log4j2Logger = LogManager.getLogger("Test-File-Logger");
-            ((org.apache.logging.log4j.core.Logger) log4j2Logger).getContext().setConfigLocation(configLocation);
+            assertThat(log4j2Logger).isInstanceOf(org.apache.logging.log4j.core.Logger.class);
+
+            LoggerContext loggerContext = ((org.apache.logging.log4j.core.Logger) log4j2Logger).getContext();
+            loggerContext.setConfigLocation(configLocation);
```

Comment:
[for reviewer] this does not help fixing the tests, but would help to identify test flakyness on windows